src/platform/graphics/graphics-device.js

```diff
@@ -104,6 +104,14 @@ class GraphicsDevice extends EventHandler {
      */
     isWebGL2 = false;
 
+    /**
+     * True if the back-buffer is using HDR format, which means that the browser will display the
+     * rendered images in high dynamic range mode. This is true if the displayFormat is set to
```

Comment:
Link `displayFormat`?

Reply:
we cannot link to options.displayFormat directly I think, but I link a function accepting this options object.